samples/conftest.py

```diff
@@ -12,11 +12,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import datetime
 import os
 
 import pytest
 
+import logging
+
+
+PYTHONWARNINGS="default"
+# import http.client as http_client
+
+# http_client.HTTPConnection.debuglevel = 3
```

Comment:
Are these meant to be in the sample, or just commented out debugging?

Reply:
They are not supposed to be in the sample, I have removed them.